projects/elife.yaml

```diff
@@ -2169,3 +2169,27 @@ microsimulation-demo:
         ports:
             1280: 80
             1281: 8081
+
+bioprotocol:
+    repo: https://github.com/elifesciences/bioprotocol-service
+    aws:
+        type: t2.micro
+        ports:
+            - 22
+            - 80
+            - 443
+        sns:
+            # endpoint for BioProtocol to publish updates to article protocol data
+            - bioprotocol-updates--{instance}
+        sqs:
+            # single queue that receives messages about:
+            # - updates to articles (internally)
+            # - updates to article protocol data (from BioProtocol)
```

Comment:
these will be wildly different messages, but if the application is smart enough

Reply:
dispatching a message based on the shape of the data seems like a pretty basic thing to do.